util/config/config.go

```diff
@@ -431,3 +436,33 @@ func (config *Configuration) CheckPorts(myIP string) (bool, error) {
 	}
 	return true, nil
 }
+
+func OpenConfigFile() ([]byte, error) {
+	var file []byte
+
+	configFile := ConfigFile
+	if configFile == "" {
+		configFile = DefaultConfigFile
+	}
+	_, err := os.Stat(configFile)
+	if err != nil {
+		return file, err
+	}
+	file, err = ioutil.ReadFile(configFile)
+	if err != nil {
+		return nil, err
+	}
+
+	// Remove the UTF-8 Byte Order Mark
+	file = bytes.TrimPrefix(file, []byte("\xef\xbb\xbf"))
+	return file, err
+
+}
+
+func WriteConfigFile(data []byte) error {
```

Comment:
It's better to accept a config as input argument, and put `bytes, err := json.MarshalIndent(&configuration, "", "    ")` part inside this function